core/crawl/crawler.py

```diff
@@ -264,14 +263,15 @@ def _main(self, argv):
         # initialize cookies
         if cookie_string:
             try:
+
                 start_cookies = self._parse_cookie_string(cookie_string)
+                for cookie_string in start_cookies:
```

Comment:
I would rename that one to avoid any confusion with the other cookie_string variable